mava/configs/env/scenario/15x15-4p-3f.yaml

```diff
@@ -0,0 +1,15 @@
+# The config of the 15x15-4p-3f scenario with the VectorObserver set as default
+task_name: 15x15-4p-3f
+
+task_config:
+  grid_size: 15
+  fov: 15
+  num_agents: 4
+  num_food: 3
+  max_agent_level: 2
+  force_coop: False
+
+# GridObserver config
+grid_obs_config:
```

Comment:
They use the vector one in the paper yes. Since we don't have any conv nets yet in mava it might even be a good idea to remove the grid obs stuff

Reply:
Yes the default observer is the vector one so these configs will be unnecessary since we're only considering the vector observation, I'll remove them.